singleuser/cuda.Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG BASE_IMAGE=renku/singleuser:latest
+FROM $BASE_IMAGE
+
+MAINTAINER Swiss Data Science Center <info@datascience.ch>
+
+# Insall cuda
+# From: https://gitlab.com/nvidia/cuda/blob/ubuntu18.04/9.2/base/Dockerfile
+USER root
+RUN apt-get update && apt-get install -y --no-install-recommends gnupg2 curl ca-certificates && \
+    curl -fsSL https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1710/x86_64/7fa2af80.pub | apt-key add - && \
+    echo "deb https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1710/x86_64 /" > /etc/apt/sources.list.d/cuda.list && \
+    echo "deb https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1604/x86_64 /" > /etc/apt/sources.list.d/nvidia-ml.list && \
+    apt-get purge --autoremove -y curl && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV CUDA_VERSION 9.2.148
```

Comment:
Make this an ARG?

Reply:
The thing is taken directly from nvidia: https://gitlab.com/nvidia/cuda/blob/ubuntu18.04/9.2/base/Dockerfile

Don't know if that's the only change with other cuda versions.

Reply:
I see, unfortunately there's a new Cuda version every few weeks